woocommerce-transbank/src/TransbankWebpayOrders.php

```diff
@@ -43,6 +45,20 @@ public static function getByToken($token)
         return $webpayTransaction;
     }
     
+    public static function getBySessionIdAndOrderId($TBK_ID_SESION, $TBK_ORDEN_COMPRA)
+    {
+        global $wpdb;
+        $transactionTableName = TransbankWebpayOrders::getWebpayTransactionsTableName();
+        $sql = $wpdb->prepare("SELECT * FROM $transactionTableName WHERE session_id = '%s' && order_id='%s'", $TBK_ID_SESION, $TBK_ORDEN_COMPRA);
+        $sqlResult = $wpdb->get_results($sql);
+        if (!is_array($sqlResult) || count($sqlResult) <= 0) {
+            throw new TokenNotFoundOnDatabaseException("Token '{$token}' no se encontró en la base de datos de transacciones, por lo que no se puede completar el proceso");
```

Comment:
it seems that the variable `$token` does not exist in this method

Reply:
👏 thanks